### PR TITLE
Add russian translation

### PR DESCRIPTION
**Pull request by DmitryScaletta**

### CODE REVIEW

**Review by DGKSK8LIFE — COMMENTED** [review]
Are there any problems with Cyrillic encoding that you've encountered? Ty.

**Comment by overlisted** [comment]
Why would there be any problems? It's all just unicode nowadays

**Comment by DGKSK8LIFE** [comment]
Yeah I just wanted to see about device/browser compatibility. Might pull his fork branch locally later for testing.

File: kofta/public/locales/ru/translation.json [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    "roomChat": {
+      "title": "Чат",
+      "emotesSoon": "[смайлы скоро]",
+      "bannedAlert": "Вы были забанены в чате",
+      "waitAlert": "Нужно немного подождать перед отправкой следующего сообщения",
+      "search": "Поиск",
+      "searchResults": "Результаты поиска",
+      "recent": "Часто используемые",
+      "sendMessage": "Отправить сообщение",
+      "whisper": "Whisper",
```
Comment (overlisted) [inline kofta/public/locales/ru/translation.json:222]: Whisper means шептать, but it's better to use Личные Сообщения in that context

**Comment by finch0212** [comment]
Чёрт, не успел сделать свой ПР

File: kofta/public/locales/ru/translation.json [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+      "title": "Запланированные комнаты",
+      "noneFound": "ничего не найдено",
+      "allRooms": "все запланированные комнаты",
+      "myRooms": "мои запланированные комнаты",
+      "scheduleRoomHeader": "Запланировать комнату",
+      "startRoom": "создать комнату",
+      "modal": {
+        "needsFuture": "needs to be in the future",
+        "roomName": "room name",
+        "minLength": "min length 2"
```
Comment (overlisted) [inline kofta/public/locales/ru/translation.json:210]: Мин. длина

File: kofta/public/locales/ru/translation.json [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+    "scheduledRooms": {
+      "title": "Запланированные комнаты",
+      "noneFound": "ничего не найдено",
+      "allRooms": "все запланированные комнаты",
+      "myRooms": "мои запланированные комнаты",
+      "scheduleRoomHeader": "Запланировать комнату",
+      "startRoom": "создать комнату",
+      "modal": {
+        "needsFuture": "needs to be in the future",
+        "roomName": "room name",
```
Comment (overlisted) [inline kofta/public/locales/ru/translation.json:209]: Название комнаты

File: kofta/public/locales/ru/translation.json [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+    "_comment": "Modules UI Internationalization Strings",
+    "scheduledRooms": {
+      "title": "Запланированные комнаты",
+      "noneFound": "ничего не найдено",
+      "allRooms": "все запланированные комнаты",
+      "myRooms": "мои запланированные комнаты",
+      "scheduleRoomHeader": "Запланировать комнату",
+      "startRoom": "создать комнату",
+      "modal": {
+        "needsFuture": "needs to be in the future",
```
Comment (overlisted) [inline kofta/public/locales/ru/translation.json:208]: Должна быть в будущем

File: kofta/public/locales/ru/translation.json [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+      "invited": "приглашён",
+      "inviteToRoom": "пригласить в комнату"
+    },
+    "micPermissionBanner": {
+      "permissionDenied": "Не удалось получить разрешение на доступ к вашему микрофону (вам, возможно, нужно зайти в настройки браузера и перезагрузить страницу)",
+      "dismiss": "отклонить",
+      "tryAgain": "попробовать снова"
+    },
+    "keyboardShortcuts": {
+      "setKeybind": "set keybind",
```
Comment (overlisted) [inline kofta/public/locales/ru/translation.json:137]: Установить комбинацию клавиш

File: kofta/public/locales/ru/translation.json [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+      "listening": "listening",
+      "toggleMuteKeybind": "toggle mute keybind",
```
Comment (overlisted) [inline kofta/public/locales/ru/translation.json:139]: Переключить звук

File: kofta/public/locales/ru/translation.json [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+      "togglePushToTalkKeybind": "toggle push-to-talk keybind"
```
Comment (overlisted) [inline kofta/public/locales/ru/translation.json:140]: Переключить push-to-talk

File: kofta/public/locales/ru/translation.json [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+      "inviteToRoom": "пригласить в комнату"
+    },
+    "micPermissionBanner": {
+      "permissionDenied": "Не удалось получить разрешение на доступ к вашему микрофону (вам, возможно, нужно зайти в настройки браузера и перезагрузить страницу)",
+      "dismiss": "отклонить",
+      "tryAgain": "попробовать снова"
+    },
+    "keyboardShortcuts": {
+      "setKeybind": "set keybind",
+      "listening": "listening",
```
Comment (overlisted) [inline kofta/public/locales/ru/translation.json:138]: Слушаю

**Review by overlisted — CHANGES_REQUESTED** [review]
your translation is trash

**Comment by benawad** [comment]
can you switch this to `staging` branch

**Comment by overlisted** [comment]
can't you do it yourself

**Comment by DmitryScaletta** [comment]
I will make a new PR to `staging` then

**Comment by DGKSK8LIFE** [comment]
chill bro @overlisted

**Pull request closed without merging**